.circleci/config.yml

```diff
@@ -157,7 +174,6 @@ jobs:
             kubectl rollout status deployment/${DEPLOYMENT} -n $CLUSTER_NAMESPACE
 
 workflows:
-  version: 2
   build_and_deploy:
     jobs:
       - build:
```

Comment:
It seems that `build` job has branch filter. If we want to run lighthouse before PR merged, would we need to move it another job ?

Reply:
Updated([a507ae0](https://github.com/twreporter/twreporter-react/pull/2227/commits/a507ae02c66253b8823814f76f74867e6f82786a)).